packages/dds/tree/src/feature-libraries/sequence-field/markListFactory.ts

```diff
@@ -18,7 +19,12 @@ export class MarkListFactory<TNodeChange> {
     private offset = 0;
     public readonly list: MarkList<TNodeChange> = [];
 
-    public constructor(private readonly moveEffects?: MoveEffectTable<TNodeChange>) {}
+    public constructor(
+        // TODO: Is there a usage of MarkListFactory where we need a non-undefined revision?
```

Comment:
Looks like `DetachedNodeTracker` does pass the revision in.

Reply:
We may want to have `DetachedNodeTracker` work with composite changes some day, but I agree that's at best a low priority.
I agree it seems weird not to keep this.

Reply:
Actually, is there ever a way for this revision to be needed (I mean in theory, beyond the current code usages)?
I'm strugling to figure that out. It seems like maybe it would be needed when rebasing a composite change: specifically when rebasing the marks for a field whose changes are not composite, contains move/return marks whose other ends are in fields that do contain marks from different revisions.

Reply:
About "in compose they always have the revision explicitly annotated on the mark":
In the compose case, wouldn't it be possible to have a scenario where one end of the move is in a trait with changes from multiple revisions (i.e., with revision info coming from marks) but the other end is in a trait that doesn't (i.e., the revision info is form context).

Either way, it seems like the comment can be removed.